career/360-review.md

```diff
@@ -0,0 +1,35 @@
+# 360 Reviews
+
+ A 360 review is a tool for gathering feedback for an individual (evaluee). It is done by interviewing people who work with the evaluee in different contexts or with different roles.
+
+## Goal
+
+The goal of the 360 review is to provide in-depth feedback for the evaluee so that they can reflect on how others perceive their work. The ultimate goal is growth, and it is achieved by reflection. It is usually the most extensive feedback that the evaluee receives in the year.
+
+
+## Preparation
+
+The preparation consists of getting to know what is the evaluee missing the most. What do they really want to know?
+Set up a short interview with enough anticipation. 1 or 2 weeks in advance. Let clear the intentions in the agenda. Make them prepare a list of questions. During the interview, dig a bit deeper into the list. Why do they want to know that? Whom do they want to hear from? The list may include a close client and their manager. Is there anything that's not on the list but may be interesting? Use your experience from other 360 reviews to suggest what they may want to know.
+
+
+## Conducting the interviews
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
+
+There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what it is "nice" about pairing with them.
```

Comment:
Would you mind curating a few resources that you've found useful so we can optimize our time?

Reply:
I'll do that. I would like to keep it outside this PR.